flow-server/src/main/java/com/vaadin/flow/dom/impl/StyleAttributeHandler.java

```diff
@@ -65,6 +59,10 @@ public void setAttribute(Element element, String attributeValue) {
         parseStyles(attributeValue).forEach(style::set);
     }
 
+    private static final char SEMICOLON = ";".charAt(0);
+    private static final char PARENTHESIS_OPEN = "(".charAt(0);
+    private static final char PARENTHESIS_CLOSED = "(".charAt(0);
```

Comment:
```suggestion
    private static final char SEMICOLON = ';';
    private static final char PARENTHESIS_OPEN = '(';
    private static final char PARENTHESIS_CLOSED = ')';
```
We don't need to generate a string here.